package edu.scsa.day5;

import java.io.FileInputStream;
import java.io.FileNotFoundException;
import java.io.FileOutputStream;
import java.io.IOException;

public class TestException {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("main start");
		FileInputStream fileIn = null;
		try {
			
//			FileOutputStream fileOut = new FileOutputStream("myData.txt");
//			fileOut.write('k');
			fileIn = new FileInputStream("myData.txt");
			int cnt = fileIn.read();
			System.out.println("Data :" +cnt);
			System.out.println("파일을 정상적으로 읽고 있습니다");
		} catch (FileNotFoundException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}catch (IOException e) {
			e.printStackTrace();
			// TODO: handle exception
		}finally {
			System.out.println("finally 실행");
			if(fileIn!=null) {
				try {
					fileIn.close();
				} catch (IOException e) {
					// TODO Auto-generated catch block
					e.printStackTrace();
				}
			}
		}
		System.out.println("main end");
		
	}

}
